web/src/main/java/pl/isa/freshmenindustries/playGame/PlayGameService.java

```diff
@@ -0,0 +1,45 @@
+package pl.isa.freshmenindustries.playGame;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import pl.isa.freshmenindustries.game.Game;
+import pl.isa.freshmenindustries.response.Response;
+
+import java.util.UUID;
+
+@Service
+@Slf4j
+public class PlayGameService {
+    private final PlayGameRepository playGameRepository;
+
+    public PlayGameService(PlayGameRepository playGameRepository) {
+        this.playGameRepository = playGameRepository;
+    }
+
+    public Response getAllPlayGames() {
+        log.info("Getting all play games list");
+        try {
+            return new Response(Boolean.TRUE, playGameRepository.getAllPlayGame());
```

Comment:
Jeżeli getAllPlayGame() jest blok try/catch łapiący ewentualny wyjątek, a nie jest rzucany żaden nowy, to raczej tutaj nie mamy co się spodziewać żadnego wyjątku